src/ManageCourses.Api/Mapping/CourseLoader.cs

```diff
@@ -53,7 +53,7 @@ public Course LoadCourse(IEnumerable<UcasCourse> courseRecords, IEnumerable<Ucas
                 returnCourse.ProgramType = organisationCourseRecord.ProgramType;
                 returnCourse.ProfpostFlag = organisationCourseRecord.ProfpostFlag;
                 returnCourse.StudyMode = organisationCourseRecord.Studymode;
-                var subjects = organisationCourseRecord.UcasInstitution.UcasCourseSubjects
+                var subjects = organisationCourseRecord.CourseCode.UcasCourseSubjects
```

Comment:
> ...CourseCode.UcasCourseSubjects

Is `CourseCode` a strangely named property because that doesn't make a lot of sense!

Reply:
Yes, yes it is, and I created it.
It's the join table I added to the ucas data to give us unique inst/course code combinations. I couldn't call it UcasCourse because that's already there and is actually a ucas-course-campus thing.

Reply:
Made a card should we want to clean up https://trello.com/c/NrOqnHQ1/377-rename-ucas-tables